blocks/blockstore/util/remove.go

```diff
@@ -0,0 +1,100 @@
+package blockstore_util
+
+import (
+	"fmt"
+	"io"
+
+	bs "github.com/ipfs/go-ipfs/blocks/blockstore"
+	"github.com/ipfs/go-ipfs/pin"
+	ds "gx/ipfs/QmbzuUusHqaLLoNTDEVLcSF6vZDHZDLPC7p4bztRvvkXxU/go-datastore"
+	key "gx/ipfs/Qmce4Y4zg3sYr7xKM5UueS67vhNni6EeWgCRnb7MbLJMew/go-key"
+	cid "gx/ipfs/QmfSc2xehWmWLnwwYR91Y8QF4xdASypTFVknutoKQS3GHp/go-cid"
+)
+
+type RemovedBlock struct {
+	Hash  string `json:",omitempty"`
+	Error string `json:",omitempty"`
+}
+
+type RmBlocksOpts struct {
+	Prefix string
+	Quiet  bool
+	Force  bool
+}
+
+func RmBlocks(blocks bs.GCBlockstore, pins pin.Pinner, out chan<- interface{}, cids []*cid.Cid, opts RmBlocksOpts) error {
+	go func() {
```

Comment:
It returns an error in future code that is not ready to be merged.  See https://github.com/ipfs-filestore/go-ipfs/blob/master/blocks/blockstore/util/remove.go#L30.